source/stylesheets/scss/furatto/_tabs.scss

```diff
@@ -0,0 +1,150 @@
+// Buttons
+
+//
+// @imports
+//
+@import "compass";
+@import "functions";
+@import "media_queries";
+@import "pseudo_selectors";
+
+//
+// @variables
+//
+
+$base-font-family: "Open Sans", "Helvetica", sans-serif !default;
+
+//Padding sizes for tabs
+$tab-padding-bottom-top: 8 !default;
+
+$tabs-margin: 10 0 20 0 !default;
+
+$tab-view-default-padding: 15px 10px;
+
+$tab-option-link-padding: 8px;
+
+$border-radius-default: 4px;
+
+$tab-border-width: 1px;
+
+// General tabs section styles
+section.tabs {
+
+  margin: $tabs-margin;
+  width: 100%;
+
+  // Tab menu list
+  .tab.list {
+    display: table;
```

Comment:
Properties should be ordered content, display, list-style, margin, padding, width